src/control/security/pem.go

```diff
@@ -38,33 +38,43 @@ import (
 )
 
 const (
-	SafeKeyPerm  os.FileMode = 0400
-	SafeCertPerm os.FileMode = 0664
-	SafeDirPerm  os.FileMode = 0700
+	MaxKeyPerm  os.FileMode = 0700
+	MaxCertPerm os.FileMode = 0664
+	MaxDirPerm  os.FileMode = 0700
 )
 
-type insecureError struct {
+type improperError struct {
```

Comment:
Might be clearer to call it badPermsError or something like that

Reply:
good idea.